sdlf-utils/ingestion-examples/preprocessing/fargate-unzip/parameters-dev.json

```diff
@@ -0,0 +1,46 @@
+[
+    {
+        "ParameterKey": "pTaskName",
+        "ParameterValue": "ingestion"
+    },
+    {
+        "ParameterKey": "pEnv",
+        "ParameterValue": "dev"
+    },
+    {
+        "ParameterKey": "pS3LandingBucketName",
+        "ParameterValue": "<landing_bucket_name>"
+    },
+    {
+        "ParameterKey": "pS3RawBucketName",

```

Comment:
Likewise, should we just reference the existing parameter? `/SDLF/S3/CentralBucket`

Reply:
Done